prac_05/emails.py

```diff
@@ -0,0 +1,22 @@
+""""
+estimate: 30 mins
+actual: 40 mins
+"""
+email_name = {}
+
+while True:
+    email = input("Email: ")
+    if not email:
+        break
+    username = email.split('@')[0]
```

Comment:
You have forgotten that emails might have "." in between names such as lindsay.ward needs to be printed as lindsay ward not lindsay.ward